{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2022-40228",
      "ASSIGNER": "psirt@us.ibm.com"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://www.ibm.com/support/pages/node/6840759",
          "name": "https://www.ibm.com/support/pages/node/6840759",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/235527",
          "name": "https://exchange.xforce.ibmcloud.com/vulnerabilities/235527",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM DataPower Gateway 10.0.3.0 through 10.0.4.0, 10.0.1.0 through 10.0.1.9, 2018.4.1.0 through 2018.4.1.22, and 10.5.0.0 through 10.5.0.2 does not invalidate session after a password change which could allow an authenticated user to impersonate another user on the system. IBM X-Force ID: 235527."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2022-11-22T19:15Z",
  "lastModifiedDate": "2022-11-22T19:17Z"
}